src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -56,22 +54,25 @@ public override void Initialize(IHostContext hostContext)
 
         public ActionDefinitionData Load(IExecutionContext executionContext, string manifestFile)
         {
-            var context = CreateContext(executionContext);
+            var templateContext = CreateContext(executionContext);
```

Comment:
I updated it from `context` to `templateContext` b/c I thought `context` has a misleading connotation and could lead to mixup with `executionContext`.